### PR TITLE
Docs: Add Component Array Property Access Example

### PR DESCRIPTION
**Pull request by pykettk**
## Description
- Adds an example solution for accessing Magewire component array property values using the magic AlpineJS `$wire` property

### CODE REVIEW

**Comment by pykettk** [comment]
@wpoortman As before, if I've misunderstood something here, please feel free to close/correct as needed 🙂

File: docs/Alpine.md [hunk 1]
```diff
@@ -23,3 +23,30 @@ class Explanation extends \Magewirephp\Magewire\Component
...
+```
+
+```html
+<div>
+    <div x-data>
+        <!-- Output 'bar' -->
+        <span x-text="$wire.foo[0]['foo']"></span>
+        
+        <!-- Output 'fooBar' -->
+        <span x-text="$wire.foo['nonNumericKey'][0]"></span>
```
Comment (wpoortman) [inline docs/Alpine.md:49]: @pykettk, did you also try $wire.foo.nonNumericKey['0']? Would love to see that in there as an example when that does work.

Comment (pykettk) [reply docs/Alpine.md:49]: @wpoortman I didn't - I'll give that a try later on and update here as required, good idea 👍 Thanks 🙂

Comment (wpoortman) [reply docs/Alpine.md:51]: I think that will work also 😅

Comment (pykettk) [reply docs/Alpine.md:53]: @wpoortman It does - I just tested to confirm 😅 I've added a couple more examples so they should now cover all bases 🤞

**Pull request merged**